frontend/public/components/marketplace/_marketplace.scss

```diff
@@ -0,0 +1,18 @@
+.co-marketplace-modal {
+  &--body {
```

Comment:
Aren't these usually `&__body {` ?

Reply:
I have seen both used a lot in console. If underscore is the standard we want to set, I am happy to change that